languages/rust/template/src/bin/pipe.rs

```diff
@@ -0,0 +1,165 @@
+// This file provides interop with the langserver
+//
+use algorithm;
```

Comment:
What compiles the `pipe` binary here? Is this compiled along side the primary algorithm project?

Reply:
Files in the `src/bin` directory automatically compile as binaries of the same name for a cargo project.

Reply:
this is not longer accurate after the latest commit. Cargo.toml names the overall project `algorithmia-pipe` now